docs/Gemfile

```diff
@@ -17,13 +17,14 @@
 ################################################################################
 
 source 'https://rubygems.org'
-ruby '~> 2.3.0'
+ruby '~> 2'
```

Comment:
When we upgraded this from `~> 1.9`, the nightly Buildbot stopped working: https://ci.apache.org/builders/flink-docs-master/builds/557/steps/Flink%20docs/logs/stdio